.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
+.php_cs.cache
```

Comment:
Instead of ignoring the cache file, we could of course just disable the cache. Let me know what you prefer.